CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+# Changelog 1.9
+
+## 1.9.0 - 2022-10-21
+
+### Added
+
+- Add the following capabilities to the Multi-cluster feature:
```

Comment:
```suggestion
- Add the following capabilities to Multi-cluster feature:
```

Reply:
"the" should be there. All change logs have it and Antonin corrected me several times.

Reply:
Thanks, got that.